reformat/fix_ft_int_issue.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
+
+
+# purpose is to attempt to fix incompatible type char/int issue when run shiny app
+    # attempt to change model name 6 --> 'model6' in 3 places
+    # 1. predictions/
+    # 2. feature-sets/
+    # tmpdir/tmp--gnosis*** obj3 writout
+
+############## obj3 writout
+ft_input = '/Users/leejor/Ellrott_Lab/gdan-tmp-webdash/data/tmpdir/tmp--gnosis_obj3.tsv'
+ft_output = '/Users/leejor/Ellrott_Lab/gdan-tmp-webdash/data/tmpdir/tmp--fixattempt2--gnosis_obj3.tsv'
```

Comment:
Hardcoding paths specific to your personal machine isn't reproducible for other people. Try to take as command line arguments.

Reply:
rm unnecessary script